lib/schemaUtils.js

```diff
@@ -1037,6 +1037,12 @@ module.exports = {
     }
 
     securitySet.forEach((security) => {
+      if (_.isEmpty(security)) {
```

Comment:
This cheks if security is empty or not, what we need is check for specifically empty object. Can you add `_.isObject(security) && _.isEmpty(security)` instead.

Reply:
Sure, I will do that.